sql: add a link in `CREATE INDEX` (#3430)
pingcap/docs: sql-statements/sql-statement-create-index.md

>
> Expression index is still an experimental feature. It is **NOT** recommended that you use it in the production environment.

> If you need to use this feature, set the following settings in [TiDB Configuration File](/tidb-configuration-file.md#allow-expression-index-New-in-v400):

Reviewer: ```suggestion
To use this feature, make the following setting in [TiDB Configuration File](/tidb-configuration-file.md#allow-expression-index-new-in-v400):
```

> If you need to use this feature, set the following settings in [TiDB Configuration File](/tidb-configuration-file.md#allow-expression-index-new-in-v400):

Reviewer: ```suggestion
To use this feature, make the following setting in [TiDB Configuration File](/tidb-configuration-file.md#allow-expression-index-new-in-v400):
```
